src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -1042,6 +1042,15 @@ enum JavaThreadState {
   _thread_max_state         = 12  // maximum thread state+1 - used for statistics allocation
 };
 
+enum LockingMode {
```

Comment:
Thank you for having named constants. 

But I'd use enum class here. Either that or prefix the names with something like "LM_", but enum class is preferred. The names are far to generic for global scope, especially since this lives in globalDefinitions.hpp.